src/views/TradingCompetition/components/YourScore/ScoreProfile.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react'
+import styled from 'styled-components'
+import { NoProfileAvatarIcon, LaurelLeftIcon, LaurelRightIcon } from '@pancakeswap-libs/uikit'
+import { Profile } from 'state/types'
+import ProfileAvatar from '../../../Profile/components/ProfileAvatar'
+import Sticker from '../Sticker'
+
+interface ScoreHeaderProps {
+  profile: Profile
+}
+
+const Wrapper = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+`
+
+const LaurelWrapper = styled.div<{ dir?: 'l' | 'r' }>`
+  transform: ${({ dir }) => (dir === 'l' ? 'rotate(30deg)' : 'rotate(-30deg)')};
```

Comment:
Please use `left` and `right`

Reply:
Oh this shit is getting ripped out now we have Laurels in the ui kit ✊